alf/algorithms/algorithm.py

```diff
@@ -1300,18 +1300,16 @@ def _train_experience(self,
             # result would be
             #
             # env_ids =   [0,  0,  0,  0,  1,  1,  1,  1]
-            # positions = [7, 10, 13, 16,  7, 10, 13, 16] (before circular())
+            # positions = [7, 10, 13, 16,  7, 10, 13, 16]
             num_envs = batch_info.env_ids.shape[0]
-            replay_buffer = batch_info.replay_buffer
             num_mini_batches_per_original_traj = length // mini_batch_length
             batch_info = BatchInfo(
                 env_ids=batch_info.env_ids.repeat_interleave(
                     num_mini_batches_per_original_traj),
-                positions=replay_buffer.circular(
-                    batch_info.positions.repeat_interleave(
-                        num_mini_batches_per_original_traj) + torch.arange(
-                            0, length, mini_batch_length).repeat(num_envs)),
-                replay_buffer=replay_buffer)
+                positions=(batch_info.positions.repeat_interleave(
```

Comment:
unnecessary '(' after '='

Reply:
Hmm I was trying to make it format better. Will remove it.